code/modules/cargo/exports.dm

```diff
@@ -147,7 +147,7 @@ Then the player gets the profit from selling his own wasted time.
 	if(amount <=0 || (the_cost <=0 && !allow_negative_cost))
 		return FALSE
 	if(dry_run == FALSE)
-		if(SEND_SIGNAL(O, COMSIG_ITEM_SOLD, item_value = get_cost(O, apply_elastic)) & COMSIG_ITEM_SPLIT_VALUE)
+		if(SEND_SIGNAL(O, COMSIG_ITEM_SOLD, get_cost(O, apply_elastic)) & COMSIG_ITEM_SPLIT_VALUE)
```

Comment:
Removing this named argument broke exporting cubes and pricetags, making a fix for it on another PR but JSYK